.travis.yml

```diff
@@ -6,6 +6,11 @@ python:
 services:
     - rabbitmq
 
+before_install:
+    - wget -O toxiproxy-server https://github.com/Shopify/toxiproxy/releases/download/v2.0.0/toxiproxy-server-linux-amd64
+    - chmod +x toxiproxy-server
+    - export PATH=$PATH:$PWD/
```

Comment:
this seems a bit dangerous (what other binaries might end up in the working dir?). probably fine for travis, but better practice might be to put this binary into a subdirectory and just add that